src/client/components/app-header/app-header.vue

```diff
@@ -15,7 +16,7 @@
           </nuxt-link>
         </li>
       </ul>
-      <language-selector :locales="$i18n.locales" />
+      <!-- <language-selector :locales="$i18n.locales" /> -->
```

Comment:
We're going to need this component. So I'd like to keep it in. I'm afraid we're going to need to put something in the design for this. Maybe start with a `<select>` element with the language options in there so it takes up the minimum amount of space.